### PR TITLE
Drop support for EOL Python <= 2.6

### PR DESCRIPTION
**Pull request by hugovk**
Resolves https://github.com/httplib2/httplib2/issues/42 which has ~11~ 12 votes for and no votes against.

### CODE REVIEW

File: python2/httplib2test.py [hunk 1]
```diff
@@ -1,10 +1,10 @@
-#!/usr/bin/env python2.4
+#!/usr/bin/env python2.7
```
Comment (temoto) [inline python2/httplib2test.py]: Just remove shebang, please.

File: python2/httplib2test.py [hunk 2]
```diff
@@ -105,7 +100,7 @@ def test(self):
         self.assertEqual(233, len(httplib2.safename(uri2)))
         self.assertEqual(233, len(httplib2.safename(uri)))
         # Unicode
-        if sys.version_info >= (2,3):
+        if sys.version_info >= (2, 7):
```
Comment (temoto) [inline python2/httplib2test.py]: For `python2/httplib2test.py` there is no other version except 2.7, so please remove this condition.

File: python2/httplib2test.py [hunk 3]
```diff
@@ -263,7 +254,7 @@ def testGetConnectionRefused(self):
         self.assertEqual(response.status, 400)
 
     def testGetIRI(self):
-        if sys.version_info >= (2,3):
+        if sys.version_info >= (2, 7):
```
Comment (temoto) [inline python2/httplib2test.py]: For python2/httplib2test.py there is no other version except 2.7, so please remove this condition.

File: Makefile [hunk 4]
```diff
@@ -19,24 +15,24 @@ release:
 	if [ $(VERSION) -ne $(INLINE_VERSION_3) ]; then exit 1; fi
 	if [ $(VERSION) -ne $(INLINE_VERSION) ]; then exit 1; fi
 
-	-find . -name "*.pyc" | xargs rm 
-	-find . -name "*.orig" | xargs rm 
+	-find . -name "*.pyc" | xargs rm
+	-find . -name "*.orig" | xargs rm
```
Comment (cdent) [inline Makefile]: unrelated change, but may as well get it at some point, so okay

**Comment by hugovk** [comment]
Updated according to review comments and in /python3/httplib2test.py for the same reason.

**Comment by hugovk** [comment]
Merge conflict resolved. 

@cdent Thanks for your review, I've made updates. How does it look now?

**Comment by temoto** [comment]
Everything that was blocking 2/3 merge is done by now. 

@hugovk sorry, this patch will not be merged as is, but instead your work will be used to construct new single code base for python2/3. Thank you again. I hope you don't feel disappointment.

**Comment by hugovk** [comment]
No problem whatsoever, glad it will be of some use!

**Pull request closed without merging**